app/jobs/payment_poll_job.rb

```diff
@@ -0,0 +1,14 @@
+class PaymentPollJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Payment.not_completed.each do |payment|
+      payment.update(status: payment.mollie_payment.status)
+      next unless payment.mollie_payment.paid?
+
+      CreditMutation.create(user: payment.user,
+                            amount: payment.mollie_payment.amount.value,
+                            description: 'Ideal inleg via Mollie', created_by: payment.user)
```

Comment:
In payments_controller.rb staat hier gewoon 'Ideal inleg' zonder 'via Mollie' erbij. Dat lijkt mij ook prima, voor de gebruiker maakt het weinig uit via welke provider de betaling nou is gedaan.